test/cases/ostree-simplified-installer.sh

```diff
@@ -125,6 +125,12 @@ SYSROOT_RO="false"
 ANSIBLE_USER="admin"
 FDO_USER_ONBOARDING="false"
 
+# Set FIPS variable default
+FIPS="${FIPS:-false}"
```

Comment:
edge-simplified-8 and edge-simplified-cs8 are failing due to fips blueprint customization unsupported in RHEL-8.x. Are we planning to support it?

Reply:
I missed one change in that file, it should be fixed now